api/src/main/java/org/openmrs/util/HandlerUtil.java

```diff
@@ -139,13 +139,7 @@ public static void clearCachedHandlers() {
 		}
 		
 		// Return the list of handlers based on the order specified in the Handler annotation
-		handlers.sort(new Comparator<H>() {
-
-			@Override
-			public int compare(H o1, H o2) {
-				return getOrderOfHandler(o1.getClass()).compareTo(getOrderOfHandler(o2.getClass()));
-			}
-		});
+		handlers.sort(Comparator.comparing(o -> getOrderOfHandler(o.getClass())));
```

Comment:
Did you intentionally do this differently from the other one? https://github.com/openmrs/openmrs-core/commit/9847f2928dd45c90d351c961a35507e814b8d874

Reply:
This sorting is simpler than in trunk-5261 so this way is's more readable 😊

Reply:
Is there any reason why you did both differently? 😊

Reply:
Only readability

Reply:
So you decided to make one less readable than the other?

Reply:
No. There is no way to simplify lambda in trunk-5261 because there are if statements. I made them both as readable as I could